app/app.js

```diff
@@ -19,23 +19,35 @@ const whoAmI = require('./routes/who-am-i');
 const port = 8888;
 
 class App extends EventEmitter {
-
     /**
      * App contructor.
      * @param {Object} options Linz options.
      * @return {Void} Starts the app.
      */
     constructor(options) {
-
         super();
 
         const app = express();
 
         this.app = app;
 
         linz.init({
-            'express': app,
-            'options': {
+            express: app,
+            options: {
+                'login middleware': {
+                    get: [linz.api.middleware.login.get],
+                    post: [
+                        linz.api.middleware.authenticate('linz-local', {
```

Comment:
@allanchau, do you think this should be available at `linz.api.middleware.login.post`?

Reply:
@smebberson I've moved this bit of code to `linz.api.middleware.login.post` now.